src/api/student.js

```diff
@@ -63,4 +64,33 @@ app.post('/', (req, res) => {
     }
     res.successJson();
 });
+
+app.patch('/rfid', (req, res) => {
+    if (!req.query.studentId) {
+        return res.failureJson('Numeric student ID required when associating a studentId and RFID');
+    }
+    if (!req.body.rfid) {
+        return res.failureJson('Rfid number required when associating a studentId and RFID');
+    }
+    let student = StudentStore.getStudentById(parseInt(req.query.studentId));
+    if(student === undefined || student === null){
+        return res.failureJson('The student ID does not exist.');
+    }
+    if(!StudentStore.isUniqueRfid(req.body.rfid)){
+        return res.failureJson('The rfid is already associated with a student.');
+    }
+    if(student){
+        let s = {
+            id: student.id,
+            name: student.name,
+            status: student.status,
+            email: student.email,
+            major: student.major,
+            rfid: parseInt(req.body.rfid)
```

Comment:
(same as in \_\_\_)

Aside, why aren't we using the `clone` function from `consus-core`??